### PR TITLE
Adds fib agents as ghost roles

### PR DESCRIPTION
**Pull request by Alberyk**
At the request of the cciaa leader.

### CODE REVIEW

File: code/modules/cciaa/cciaa_items.dm [hunk 1]
```diff
@@ -277,3 +277,6 @@
 	storage_slots = 8
 	max_storage_space = 16
 
+/obj/item/storage/lockbox/cciaa/fib
+	name = "FIB agent briefcase"
+	desc = "A smart looking id locked briefcase"
```
Comment (Geevies) [inline code/modules/cciaa/cciaa_items.dm]:
```suggestion
	desc = "A smart looking ID locked briefcase."
```

File: code/datums/outfits/outfit_admin.dm [hunk 2]
```diff
@@ -135,7 +135,7 @@
 	back = /obj/item/storage/backpack/satchel
 	gloves = /obj/item/clothing/gloves/white
 	shoes = /obj/item/clothing/shoes/laceup
-	l_ear = /obj/item/device/radio/headset/heads/hop
+	l_ear = /obj/item/device/radio/headset/ert/ccia
```
Comment (Geevies) [inline code/datums/outfits/outfit_admin.dm]: This probably needs to be changed to "agent" headset or something.

File: code/modules/ghostroles/spawner/human/admin.dm [hunk 3]
```diff
@@ -146,6 +146,47 @@
...
+	respawn_flag = null
+
+	mob_name = null
+	mob_name_prefix = "S/Agt. "
+	mob_name_pick_message = "Pick a name."
+
+/datum/ghostspawner/human/admin/fib/escort
+	short_name = "fibescort"
+	name = "FIB Escort"
+	desc = "Protect the agents of the Federal Investigations Bureau while on field."
```
Comment (Geevies) [inline code/modules/ghostroles/spawner/human/admin.dm]:
```suggestion
	desc = "Protect the agents of the Federal Investigations Bureau while on the field."
```

File: code/datums/outfits/outfit_admin.dm (hunk 2 above)
Comment (Alberyk) [reply code/datums/outfits/outfit_admin.dm]: This was the headset that the cciaa used for a while, that was changed to be the hop one by accident or for whatever reason.

**Pull request merged**